encfs/FileUtils.cpp

```diff
@@ -1739,6 +1739,12 @@ void unmountFS(const char *mountPoint) {
   // fuse_unmount does not work on Mac OS, see #428
   unmount(mountPoint, MNT_FORCE);
 #endif
+#ifdef __CYGWIN__
+  if(fork() == 0)
```

Comment:
I think parent should way its child, because parent exits before `pkill` completes, and it makes some tests to fail.
I will then correct this.

Reply:
Done